src/components/tiles/TilesList.tsx

```diff
@@ -38,6 +41,22 @@ export function TilesList({ styles: instanceStyles }: Props) {
 
   return (
     <ScrollView testID={tID('tiles-list')} style={styles.scrollView}>
+      {pillarsTileEnabled && (
+        <PillarsTile
+          onOpenDetails={(tracker, valuesContext) => {
+            const screenName =
+              pillarSettings?.advancedScreenTrackers?.includes(
+                tracker.metricId || tracker.id,
+              ) || true
```

Comment:
I assume this was for debugging.
```suggestion
              )
```

Reply:
Yes, removed.